podman_compose.py

```diff
@@ -2352,7 +2341,7 @@ def _init_global_parser(parser):
         parser.add_argument("-v", "--version", help="show version", action="store_true")
         parser.add_argument(
             "--in-pod",
-            help="pod creation",
+            help="should podman create a default pod, custom one or not use pod at all",
```

Comment:
Please mention what values cause each of the three possibilities to be selected.